esp/services/WsDeploy/WsDeployService.cpp

```diff
@@ -4665,7 +4665,13 @@ bool CWsDeployFileInfo::handleEspServiceBindings(IEspContext &context, IEspHandl
         if (resource)
           xpath.appendf("[@resource='%s']", resource);
 
-        IPropertyTree* pSubType = pEnvRoot->queryPropTree(xpath.str());
+        IPropertyTree* pSubType = nullptr;
+        Owned<IPropertyTreeIterator> iterSubType = pEnvRoot->getElements(xpath.str());
```

Comment:
If we only care about the first entry, why are users allowed to enter more than 1 value.  Isn't this fixing the symptom of the problem and not the problem?  If more than 1 entry is allowed, but this piece of code always works on just the first entry, then you may want to just get the first entry and not create an iterator.  i.e -
if (iterSubType->first() && iterSubType->isValid())
   pSubType = &(iterSubType->query()